webapi/src/main/scala/org/knora/webapi/util/PermissionUtilV1.scala

```diff
@@ -97,16 +96,16 @@ object PermissionUtilV1 {
     /**
       * Determines the permissions that a user has on a `knora-base:Value`, and returns a permissions code in Knora API v1 format.
       *
-      * @param valueProps  a [[ValueProps]] containing the permission-relevant predicates and objects
-      *                    pertaining to the value, grouped by predicate. The predicates must include
-      *                    [[org.knora.webapi.OntologyConstants.KnoraBase.AttachedToUser]], and should include
-      *                    [[org.knora.webapi.OntologyConstants.KnoraBase.AttachedToProject]]
-      *                    and [[org.knora.webapi.OntologyConstants.KnoraBase.HasPermissions]]. Other predicates may be
-      *                    included, but they will be ignored, so there is no need to filter them before passing them to
-      *                    this function.
+      * @param valueProps     a [[ValueProps]] containing the permission-relevant predicates and objects
+      *                       pertaining to the value, grouped by predicate. The predicates must include
+      *                       [[org.knora.webapi.OntologyConstants.KnoraBase.AttachedToUser]], and should include
+      *                       [[org.knora.webapi.OntologyConstants.KnoraBase.AttachedToProject]]
+      *                       and [[org.knora.webapi.OntologyConstants.KnoraBase.HasPermissions]]. Other predicates may be
+      *                       included, but they will be ignored, so there is no need to filter them before passing them to
+      *                       this function.
       * @param subjectProject if provided, the `knora-base:attachedToProject` of the value. Otherwise, this predicate
```

Comment:
I thought we got rid of project assignments to values. 

As stated in `TransformData.scala`: 
> '$ValueProjectTransformationOption' (removes 'knora-base:attachedToProject' from values)

see also <https://github.com/dhlab-basel/Knora/blob/ca51f099e50551ab2c692453a1e6b3210099060d/webapi/src/main/scala/org/knora/webapi/util/PermissionUtilV1.scala#L119>